src/oc_reporter.erl

```diff
@@ -31,6 +32,11 @@
 
 -include("opencensus.hrl").
 
+%% behaviour for reporters to implement
```

Comment:
Those should have a few words of documentation to what they do (or expect to do)